README.md

```diff
@@ -38,53 +39,62 @@ On to the questions and answers.
 
 What about branches?
 --------------------
+
 We don't want to conflict with OpenSSL branch names. Our current plan is to append
 `+quic`. Release tags are likely to be the QUIC branch with `-releaseX` appended.
 For example, the OpenSSL tag `openssl-3.0.0-alpha12` would have a branch named
 `openssl-3.0.0-alpha12+quic` and a release tag of `openssl-3.0.0-alpha12+quic-release1`
 
 How are you keeping current with OpenSSL?
 -----------------------------------------
+
 (In other words, "What about rebasing?")
 
 Our plan is to always rebase on top of an upstream release tag. In particular:
+
 - The changes for QUIC will always be at the tip of the branch -- you will know what
-is from the original OpenSSL and what is for QUIC.
+  is from the original OpenSSL and what is for QUIC.
 - New versions are quickly created once upstream creates a new tag.
 - The use of git commands (such as "cherry") can be used to ensure that all changes
-have moved forward with minimal or no changes. You will be able to see "QUIC: Add X"
-on all branches and the commit itself will be nearly identical on all branches, and
-any changes to that can be easily identified.
+
+have moved forward with minimal or no changes. You will be able to see
+QUIC: Add X" on all branches and the commit itself will be nearly identical on all
```

Comment:
We seem to have unbalanced quotes here, now.

Reply:
Whoops